app/src/main/java/ca/carleton/gcrc/n2android_mobile1/cordova/PluginActions.java

```diff
@@ -446,6 +446,16 @@ private JSONObject updateDocumentGeometry(JSONObject doc, Location location) {
             JSONObject locationObject = new JSONObject();
             locationObject.put("wkt", "MULTIPOINT((" + longitude + " " + latitude + "))");
             locationObject.put("nunaliit_type", "geometry");
+
+            JSONArray bbox = new JSONArray();
+            bbox.put(longitude);
+            bbox.put(latitude);
+            bbox.put(longitude);
+            bbox.put(latitude);
```

Comment:
Why are you doing this twice?

Reply:
Its a bounding box and requires 4 points (x1,y1,x2,y2)